### ISSUE TITLE
Dropdown/DynamicSelectModel not listening to keyboard triggered changes

### ISSUE THREAD

**Issue by Aarkon** [udos86/ng-dynamic-forms#471]
Hey there!

First of all: Thanks for the great work so far. 👍 

Currently, I'm trying to get a dropdown menu to work that looks like this:

```
new DynamicSelectModel<string>(
    {
        id: 'stateSelect',
        label: 'Status',
        multiple: false,
        filterable: false,
        options: [
            {
                label: 'Neu',
                value: 'NEW'
            },
            {
                label: 'Aktiviert',
                value: 'ACTIVATED'
            },
            {
                label: 'Deaktiviert',
                value: 'DEACTIVATED'
            }
        ]
    },
    {
        element: {
            label: 'ui-widget'
        }
    }
)
```

As far as I can see, there is nothing unusual here, and it works pretty well except for one aspect: If I change the selection using the keyboard, the value of the form does not change. I checked this by monitoring the forms values like so:

`<pre>{{editForm.value | json}}</pre>`

I can see the selected value change when I use the mouse, but not with the keyboard. I don't know if this behaviour is a bug or I'm using the DynamicSelectModel the wrong way, but it seems counterintuitive.

Is there a workaround or a suggestion what to change? Any help would be appreciated.

**Comment by udos86** [comment]
@Aarkon Hi!

> First of all: Thanks for the great work so far. 👍

Thank you very much!

>  As far as I can see the selected value change when I use the mouse, but not with the keyboard. I don't know if this behaviour is a bug or I'm using the DynamicSelectModel the wrong way, but it seems counterintuitive.

Could you please tell me which UI package you are using? 

I'll have a look as soon as possible.

**Comment by Aarkon** [comment]
> Could you please tell me which UI package you are using?

Oh, of course: I'm on PrimeNG, and in case it is important: There is also some usage of Bootstrap in the project (not explicitly in the form above, though).

**Comment by udos86** [comment]
> I'm on PrimeNG, and in case it is important

Thanks, you're right! Prime NG dropdown does not seem to behave correctly when being selected with keyboard.

I'll have a closes look on that.

**Comment by udos86** [comment]
Should be fixed by `5.0.0`
